### PR TITLE
Update GitHub templates

### PR DESCRIPTION
**Pull request by it-praktyk**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,14 +1,30 @@
...
+If you would like to submit a bug report, please provide the output of a code provided below.
+
+If you would like write about anything else - like a feature request - feel free to remove a provided template text.
+
+Operating System, Pester version, and PowerShell version:
 
 ```powershell
 $bugReport = &{
-    $p = Get-Module -ListAvailable Pester
+    $p = get-module pester
```
[the remark this answers is not in the document]
Comment (it-praktyk) [reply .github/ISSUE_TEMPLATE.md]: I really don't like the current output which contains multi-version information if they exist.

Maybe can we add additional information 'Please be sure that the correct Pester module version is imported'?

or just select the newest
```powershell
$p = Get-Module -Name Pester | Select-Object -First 1
```

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -17,46 +33,45 @@ $bugReport
...
 
 ## 4.Current Behavior
 
+<!--
+
 If describing a bug, tell us what happens instead of the expected behavior.
 
-If suggesting a change/improvement, explain the difference from current behavior
-What is an example of the current behavior to reproduce.
+If suggesting a change/improvement, explain the difference between the current behavior.
```
[the remark this answers is not in the document]
Comment (it-praktyk) [reply .github/ISSUE_TEMPLATE.md]: OK, I agree.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nohwnd) [reply .github/ISSUE_TEMPLATE.md]: That code snippet will still fail if the module is not imported. It should do `-ListAvailable`.

[the remark this answers is not in the document]
Comment (it-praktyk) [reply .github/ISSUE_TEMPLATE.md]: OK, you are right. But Do you accept it if I will add the `-ListAvailable` parameter?

**Pull request merged**